app/actions/views/right_menu_drawer.js

```diff
@@ -6,14 +6,12 @@ import {batchActions} from 'redux-batched-actions';
 import {NavigationTypes} from 'app/constants';
 import Routes from 'app/navigation/routes';
 
-export function goToSelectTeam() {
+export function goToModalSelectTeam() {
```

Comment:
seems to me that this action is duplicate? https://github.com/mattermost/mattermost-mobile/pull/220/files#diff-fce60a2aaea1c6f085940299ea09b5bfR50